Kinvey/KinveyTests/RealtimeAclTestCase.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  RealtimeAclTestCase.swift
+//  Kinvey
+//
+//  Created by Victor Hugo on 2017-08-25.
+//  Copyright © 2017 Kinvey. All rights reserved.
+//
+
+import XCTest
+import Kinvey
+
+class RealtimeAclTestCase: KinveyTestCase {
+    
+    func testLiveStreamAclConsturctorNoParams() {
+        let acl = LiveStreamAcl()
+        XCTAssertNotNil(acl)
```

Comment:
Also, when the constructor is used without parameters, what should be the default values of the properties of the class? Is it worth asserting those?
What about usage of this class? Is that tested anywhere?

Reply:
no, we don't have tests for ACL for Realtime yet, the only reason for this test is to make sure that a constructor without any parameters is available since it should be the most common use case. Our devcenter has an example that demonstrates this usage: https://devcenter.kinvey.com/ios/guides/live-service#DirectedCommunication